src/config/tests/mod.rs

```diff
@@ -78,6 +78,11 @@ fn build_usize_test_cases() -> Vec<USizeEnvValue> {
             env_key: String::from("TX_QUEUED_CACHE_DURATION"),
             generator: Box::new(super::tx_queued_cache_duration),
         },
+        USizeEnvValue {
+            expected_default: super::token_cache_size_count(),
```

Comment:
This is testing more the behaviour of `token_cache_size_count` and not the default itself which might not be what we want.

Shouldn't this test be: "Given that `TOKEN_CACHE_SIZE_COUNT` is empty the value returned by the function is `20000`"?